### PR TITLE
[ContentBundle] Missing spec and configuration

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/ContentBundle/spec/Doctrine/ODM/PHPCR/StaticContentRepositorySpec.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace spec\Sylius\Bundle\ContentBundle\Doctrine\ODM\PHPCR;
+
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Doctrine\ODM\PHPCR\Mapping\ClassMetadata;
+use Doctrine\ODM\PHPCR\UnitOfWork;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class StaticContentRepositorySpec extends ObjectBehavior
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Doctrine/ODM/PHPCR/StaticContentRepositorySpec.php]: @author block missing

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/ActionBlockTypeSpec.php [hunk 2]
```diff
@@ -20,8 +20,11 @@ function it_is_initializable()
 
     function it_builds_a_form(FormBuilderInterface $builder)
     {
-        $builder->add('id', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('actionName', 'text', Argument::type('array'))->willReturn($builder);
+        $builder->add('id', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('actionName', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishable', null, Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishStartDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishEndDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/ActionBlockTypeSpec.php]: shouldBeCalled() - unneccesaary 2 times

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/ImagineBlockTypeSpec.php [hunk 3]
```diff
@@ -2,15 +2,16 @@
 
 namespace spec\Sylius\Bundle\ContentBundle\Form\Type;
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/ImagineBlockTypeSpec.php]: License missing

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/ImagineBlockTypeSpec.php [hunk 4]
```diff
@@ -2,15 +2,16 @@
...
 
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
 use Symfony\Component\Form\Test\FormBuilderInterface;
 
 class ImagineBlockTypeSpec extends ObjectBehavior
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/ImagineBlockTypeSpec.php]: @author block missing

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/MenuBlockTypeSpec.php [hunk 5]
```diff
@@ -20,9 +20,12 @@ function it_is_initializable()
...
     {
-        $builder->add('id', 'text',  Argument::type('array'))->willReturn($builder);
-        $builder->add('name', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('body', 'textarea', Argument::type('array'))->willReturn($builder);
+        $builder->add('id', 'text',  Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('name', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('body', 'textarea', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishable', null, Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishStartDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishEndDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/MenuBlockTypeSpec.php]: shouldBeCalled() - two times

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/ReferenceBlockTypeSpec.php [hunk 6]
```diff
@@ -20,9 +20,12 @@ function it_is_initializable()
...
     {
-        $builder->add('id', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('title', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('body', 'textarea', Argument::type('array'))->willReturn($builder);
+        $builder->add('id', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('title', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('body', 'textarea', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishable', null, Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishStartDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishEndDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/ReferenceBlockTypeSpec.php]: shouldBeCalled() - two times

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/SimpleBlockTypeSpec.php [hunk 7]
```diff
@@ -20,10 +20,13 @@ function it_is_initializable()
...
-        $builder->add('name', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('title', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('body', 'textarea', Argument::type('array'))->willReturn($builder);
+        $builder->add('parentDocument', null, Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('name', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('title', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('body', 'textarea', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishable', null, Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishStartDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishEndDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/SimpleBlockTypeSpec.php]: shouldBeCalled() - two times

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/SlideshowBlockTypeSpec.php [hunk 8]
```diff
@@ -20,10 +20,13 @@ function it_is_initializable()
...
-        $builder->add('name', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('title', 'text', Argument::type('array'))->willReturn($builder);
-        $builder->add('children', 'collection', Argument::type('array'))->willReturn($builder);
+        $builder->add('parentDocument', null, Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('name', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('title', 'text', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('children', 'collection', Argument::type('array'))->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishable', null, Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishStartDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
+        $builder->add('publishEndDate', 'datetime', Argument::type('array'))->shouldBeCalled()->shouldBeCalled()->willReturn($builder);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/SlideshowBlockTypeSpec.php]: Again :)

File: src/Sylius/Bundle/ContentBundle/spec/Form/Type/StaticContentTypeSpec.php [hunk 9]
```diff
@@ -0,0 +1,42 @@
+<?php
+
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ContentBundle/spec/Form/Type/StaticContentTypeSpec.php]: License and @author block

**Comment by aRn0D** [comment]
@michalmarcinkowski done!

**Comment by pjedrzejewski** [comment]
Thank you Arnaud! :+1: :+1: :+1:

**Pull request merged**